packages/x-telemetry/src/config.ts

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable no-underscore-dangle */
+import { ponyfillGlobal } from '@mui/utils';
+
+interface TelemetryEnv {
+  NODE_ENV?: string;
+  IS_COLLECTING: boolean | null;
+  DEBUG: boolean;
+}
+
+let cachedEnv: TelemetryEnv | null = null;
+
+function getMuiXEnabledEnvs() {
+  return (
+    process.env.MUI_X_TELEMETRY_ENABLED ||
+    process.env.REACT_APP_MUI_X_TELEMETRY_ENABLED ||
+    process.env.NEXT_PUBLIC_MUI_X_TELEMETRY_ENABLED ||
+    process.env.GATSBY_MUI_X_TELEMETRY_ENABLED ||
+    process.env.PUBLIC_MUI_X_TELEMETRY_ENABLED
+  );
+}
+
+function getMuiXDisabledEnvs() {
+  return (
+    process.env.MUI_X_TELEMETRY_DISABLED ||
+    process.env.REACT_APP_MUI_X_TELEMETRY_DISABLED ||
+    process.env.NEXT_PUBLIC_MUI_X_TELEMETRY_DISABLED ||
+    process.env.GATSBY_MUI_X_TELEMETRY_DISABLED ||
```

Comment:
OK, got it, I guess it's for https://www.gatsbyjs.com/docs/how-to/local-development/environment-variables/#accessing-environment-variables-in-the-browser

> To expose a variable in the browser, you must preface its name with GATSBY_